src/main/java/core/basesyntax/service/impl/CreateReportImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.CreateReport;
+import java.util.List;
+
+public class CreateReportImpl implements CreateReport {
+    private static final String REPORT_TITLE = "fruit,quantity";

```

Comment:
Lets initialize header with constructor, so we can use this class to create reports with different headers